cdk/src/functions/notification_opsItem_created/index.py

```diff
@@ -0,0 +1,38 @@
+import boto3
+import json
+import os
+
+client = boto3.client('sns')
+
+def log(msg):
+    print(json.dumps(msg), flush=True)
+
+def handler(event, _):
+    response_elements = event['detail']['responseElements']
+    id = response_elements.get('OpsItemId', response_elements.get('opsItemId'))
+    request_parameters = event['detail']['requestParameters']
+    desc = request_parameters.get(
+        'Description', request_parameters.get('description'))
+    title = request_parameters.get('Title', request_parameters.get('title'))
+    assert id and title and desc
+
+    url = "https://{}.console.aws.amazon.com/systems-manager/opsitems/{}".format(
+        os.environ['AWS_REGION'], id)
```

Comment:
You can also use f-strings here. I don't know if we have a linter like flake8 in place. If so we need the `# noqa` to disable it because the line exceeds 80 characters. 
```suggestion
    url = "https://{os.environ['AWS_REGION']}.console.aws.amazon.com/systems-manager/opsitems/{id}"  # noqa
```

Reply:
Alternatively, you can split a string across multiple lines with a backslash 
```python
url = f"https://{region}.console.aws.amazon.com" \
      f"/systems-manager/opsitems/{id}"
```

Reply:
I would also recommend not naming any variables `id` since this is a build-in python function